students/emorychristenson/lesson02/series.py

```diff
@@ -0,0 +1,73 @@
+def fibonacci(n):
+    """Calculates nth integer in the Fibonacci series starting from index 0."""
+    if n<0:
```

Comment:
You'll want spaces around the operators, here and elsewhere in the code:  n < 0, n == 0, etc.